[GITHUB-APPS] Roles and permissions guides (#8260)
circleci/circleci-docs: jekyll/_cci2/manage-roles-and-permissions.adoc

---
contentTags:
  platform:
  - Cloud
---
= Manage roles and permissions
:page-layout: classic-docs
:page-liquid:
:page-description: How-to guides for managing roles and permissions in CircleCI.
:icons: font
:experimental:

NOTE: If you authenticated your CircleCI account with xref:gitlab-integration#[GitLab] or xref:github-apps-integration#[GitHub apps], the content on this page applies to you.

Manage user access to organizations and projects with CircleCI roles and associated permissions. The guides on this page show how to add people to your organization, update permissions, and assign specific project roles to people in your organization. For an overview of roles and permissions in CircleCI, see the xref:roles-and-permissions-overview#[Roles and permissions overview] page.

> [#prerquisites]

Reviewer: ```suggestion
[#prerequisites]
```

> == Prerquisites

Reviewer: ```suggestion
== Prerequisites
```

* A CircleCI account connected to your code. You can link:https://circleci.com/signup/[sign up for free].
* At least one project created in your organization. See the xref:create-project#[Create a project] page for instructions.

[#add-people-to-your-organization]
== Add people to your organization

NOTE: To do this, you must have the _Organization Admin_ role.

. In the CircleCI web app, click on **Organization Settings** in the sidebar
. Click on **People**
. Click btn:[Invite]
. Enter the person's email address. You can add multiple people separated by a space too

> . Select a role from the dropdown menu to be applied to the people you're inviting

Reviewer: ```suggestion
. Select a role from the dropdown menu to be applied to the people you are inviting
```

NOTE: To do this, you must have the _Organization Admin_ role.

. In the CircleCI web app, click on **Organization Settings** in the sidebar
. Click on **People**
. Scroll to locate the person you want to update permissions for
. Use the dropdow menu next to their name to select a new role

[#remove-people-from-your-organization]
== Remove people from your organization

NOTE: You must have at least one org administrator. If you try to remove the last org administrator, you will get an error.

NOTE: To do this, you must have the _Organization Admin_ role.

. In the CircleCI web app, click on **Organization Settings** in the sidebar
. Click on **People**
. Scroll to locate the person you want to remove
. Click the elipsis (...) button next to their name and select btn:[Remove from organization]

[#assign-a-project-role]
== Assign a project role

image::{{site.baseurl}}/assets/img/docs/gl-ga/gitlab-project-settings-project-roles.png[Project roles setup page]

. In the CircleCI web app, click on **Dashboard** in the sidebar
. Select your project using the filter dropdown menu
. Click btn:[Project Settings]
. Click **People** in the sidebar menu
. Click btn:[Assign Project Roles]

> . Select the username of the Person you want to assign a role to

Reviewer: ```suggestion
. Select the username of the person you want to assign a role to
```
